tagbase_server/tagbase_server/test/test_ingest.py

```diff
@@ -38,13 +41,18 @@ def test_get_dataset_properties(self, mock_open):
             referenced_track_included,
             content,
             metadata_content,
-            processed_lines,
+            number_global_attributes_lines,
         ) = pu.get_dataset_properties("test_file")
 
         assert ptt, 117464
+        assert instrument_name, "s"
         assert metadata_content
         assert expected_property, metadata_content[0]
-        assert len(content) == 0, "no content is expected"
+        assert len(content), 2
+        assert len(metadata_content), 2
+        # TODO we use zero based indexing
+        assert number_global_attributes_lines, 1
+        assert number_global_attributes_lines != 4, "# of global attributes needs to be different to # of file lines"
```

Comment:
this is the unit test where we check that the # of global attributes processed needs to be different to the # of total lines in the file